kobocat/onadata/apps/usermodule/models.py

```diff
@@ -118,7 +118,7 @@ def __str__(self):
 
 #branch
 class Branch(models.Model):
-    id = models.BigIntegerField(null=False)
+    # id = models.BigIntegerField(null=False)
```

Comment:
Why is this being deleted?

Reply:
It is an auto-generated [field](https://docs.djangoproject.com/en/4.2/topics/db/models/#automatic-primary-key-fields). If it is kept there it will give errors while making the migration. We will migrate bh_module only, not the usedmodule. so there will be no impact on usermodule. For this, we can not make the migration.